thermo/EquationsOfState.ipynb

```diff
@@ -0,0 +1,414 @@
8vh9w8uHs8jLtnLA3+eAdz7ruIHrvWk5kDndbAsqxYHucCOtCQPRzkUT7layrWX7YRHWjPvQB8zijWMafsA2q9CHGdIX8z7LgatIQkQMTNslfvVNj8BfzrgeJl6ifD8Nfd8tBBsPLrQrubde1K29tvJ+/gQab07Ut+XsXHfQ6FmPh4Euu5ZyLy8/KY1KsXfz8G+iVbghwipXWx8Lcg7w/jr24NIjhZSLjGQgY49ZcEeQd5ZfULMyF1Km7a9R+YTr71B48MGxv56HDhQPj1TTayRZhFR8Gfe8Gb/SA+Bh7ccQWrGrQl5ofvmHz3BfTb+QM78uGSTJi2O5pHOI8LOIn95JLOHJawqULXaUoXWnMboCxiOD9RxsNxEgt1JkJ0U8hZDLv6l5wsVKsFN0yE6DhYNAK+mlS8zG+ugFt+C9nZcNfN7t3n0hdeoGajRmxYsIBFr4a5f2UZdq5dy09ffsna2bMZ1b79UT1esiXIQTRhwqFxL5s1e6XQegF/grxyJTRtCuEYKvCXyULCmCAXTBYSzhbkOlSjPgnsJ5cfCePNmkLqcwa1SCGbnWyglP52JvQKjY38eKRrUzWJwNWvw0nnwM51MOFqyLXZPkNlwnxo+geI6uu9C3w6EOIa1OfsdjNZfNK5RK3PJL3vBZy17luyFXptgMf2rGel1yXsIPk8zVw+Y32Frnka19Ccnii5fMZTTOFWJtGTD7mdTGYXLhxVH+pMBkmE/eNhn9dFImsC/NQUNkW599oroNtLbt/7/WD76sLn+XwC1P831AK+/gIe7FVod7Xatek+wvV7nvHAA/xcV6BNU3incMLx1YQJDG/alCFRUQxv2pSvQpyQ1G3WjN8uWULjc85h17p1jOnUiS/Gjw/pNY+UJchBMmHCV/Tvf2j2pPXrd9O//9RiSfJHhSfBITMT+vcPfZIcyemmw9kHGQ51s7AZ9SJHkF8mDlltM+tFjn9s5H0vQ87R21pzXIut5ka2SDoJ1s2DyaW0HJpKmTAf+o+DzO2guPf+4+D7LbD4cah7cW0ufu1jZre5iMbbNvLRPRdy1uovaVgrky+qL2Ubh37WzUd5ngV8TPkjLwjCr+lDPVqSTw7Z7ASULLaylL8WT5Jjfw1JY93ynodg18OuNTk/E1D3vqs/tE5yD3tm73bjIxd8sfp8gvs7lLUeOgMCvD0Vxhf+EnzawT20SogmV5Wpe0DXZ8L9/X9Jkr+aMIGp/fuzKzMTVNmVmcnU/v1DniQX9EsuGC958q238u/f//6o65dsCXKQDB48g6yswn+4WVk5DB48o9C2YcOKH5uVBYMHh7J2bkB1CO9006fEur9gmTmQHcYRv5p7CfJqe1AvogKkEkMC2/mWrAr26TMhENsOqt8H5MPeFyNdm6qr5olw8xRvWuE3Yf3CSNfouDP4Pcgq0iCTddBtb1wHcjvB3qSadHvuX0zvcCkNdm4l45k+nNHgK2KiivfTVeAvLOEDShhNogghqsSHkvPI5mtKaCFN6AU1HgXyIet5KDYWcxbsfRSuGgV1ToaNS2H6QLfrk8FuUhGAE4E2XmXTn4E9vlawpwdzWbU8EgXW5sKybGB/FjztEo4ZgweTk1X4ujlZWcwIdUKC65d85ahRdB85kqjYWBa+/DLLvIcLjxZS1oQWx7r27dvrkiVLwnKtqKghJTYIiEB+/hO+ciU3HLhyoavfhp9hfw40qg3V40N3naIW74dArBtepwIzcQfFdvazm2xSqEW0fQeMqE0spjankIBNWhFR+XsgazRUHwASV355EzorJrsPgTOujnRNjjtRfSnxyRMB8sdA1IpD++MOZvPKy/fx57SBnNFlaZmfT+fSiIc5jyjK/hCbRE9Kq0EvPii+WfNhx1WQPbX4voKaN8yHDYshoxPUawF3LYQhNQtfJw+YDOwAJv0HLuzitie7hOPrbHh3LzSPhRtrgkQJbM1nSBkJyROhTEiKWDd3LktGjOCqsWOJiokJ23ULiMhSVW1fdHv4a3KcSklJIjOz+BiKKSlJRdZdt4ri5UJVM6dJ3dCevzQdIjDjcD0SqGdTHR8VGtIh0lUwAFE1ocb9ka6FATj9qkjX4LiVUs91qyhpO0BKDGR6sx4fjIvnzj/8DYAWuSuJiy0+m14yidzNWZxJg3KTY4BE6pf4a1ki9Us+QKKg9t9hS3PQEh7ejPISgyYd4NYPIeU8iKsOtVPcTI0FooEuQM1Gh5JjgMYpsCGTM+Igtia0iPUaqhq78yalpLjuFUUkhTohKSKlUydSOnUK6zUrwprXgmTo0C4kJsYW2paYGMvQoV2KlIPERIqUc9uNMcYYc2SGXgOJRX4gSYxz2wGGNoDEInluosCFB1sRT3Sh7fFEcwutaEdDYovsK00rbiGawj/RRhNPK24p/aCoWlDrJaBIYkAi1PQlBqd2dckxQNehrquOX3Ii3PBc4W2PDoWERETgtDgvOU5IdNuBLkOHElskIYlNTKSLJSSAJchBk5bWmoyMHgQCSYhAIJBERkYP0tJaFykHGRkQCOCVc+tpaRGquDHGGHMcSOsIGbdBoJ7rVhGo59bTOnr7a0NGQwjEePtj3Pqg6gEG0I5kEhFcy/EA2pFK4LCuHyCVdgwgkWRASCSZdgwgQGrZByamQVIGRAUAce9JGW57SdqkwVUZUNsrXzvg1tsUKX9tGgzLgCZewtEk4NavdeVap6XRIyODJC8hSQoE6JGRQWtLSADrg2yMMcYYY6qo0vogWwuyMcYYY4wxPpYgG2OMMcYY42MJsjHGGGOMMT6WIBtjjDHGGONjCbIxxhhjjDE+liAbY4wxxhjjYwmyMcYYY4wxPpYgG2OMMcYY42MJsjHGGGOMMT7H9Ux6IrIVyAzzZesD28J8zarOYh5+FvPIsLiHn8U8/Czm4VeVYx5Q1eSiG4/rBDkSRGRJSVMWmtCxmIefxTwyLO7hZzEPP4t5+FnMi7MuFsYYY4wxxvhYgmyMMcYYY4yPJcjBlxHpClRBFv
```

Comment:
Please do not ever, ever, ever, use the jet color map. See here: https://www.kennethmoreland.com/color-advice/BadColorMaps.pdf and you can search for more references. In this case, the default color map is suitable, since you only care about distinguishing each line, right?

Reply:
Changed to the default map.